blazor/dropdown-list/data-binding.md

```diff
@@ -9,66 +9,890 @@ documentation: ug
 
 # Data Binding in Blazor DropDown List Component
 
-Data binding can be achieved by using the `bind-Value` attribute and it supports string, int, Enum and bool types. If component value has been changed, it will affect all the places where you bind the variable for the **bind-value** attribute.
+The DropDownList loads the data either from local data sources or remote data services using the [DataSource](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_DataSource) property. It supports the data type of `array` or [DataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html).
 
-```cshtml
-@using Syncfusion.Blazor.DropDowns
+The **TValue** used to specifies the type of the type parameter TValue for the DropDownList component. The **TItem** used to specifies the type of the type parameter TItem for the DropDownList component. The 
+
+The DropDownList also supports different kinds of data services such as OData, OData V4, and Web API, and data formats such as XML, JSON, and JSONP with the help of [DataManager](https://help.syncfusion.com/cr/aspnetcore-js2) adaptors.
+
+| Fields | Type | Description |
+|------|------|-------------|
+| Text |  `string` | Specifies the display text of each list item. |
+| Value |  `int or string` | Specifies the hidden data value mapped to each list item that should contain a unique value. |
+| GroupBy |  `string` | Specifies the category under which the list item has to be grouped. |
+| IconCss |  `string` | Specifies the icon class of each list item. |
+
+> When binding complex data to the DropDownList, fields should be mapped correctly. Otherwise, the selected item remains undefined.
+
+## Binding local data
 
-<p>DropDownList value is:<strong>@DropVal</strong></p>
+You can bind the local data to the DropDownList component as mentioned in the below way.
```

Comment:
Change the sentence You can bind the local data to the DropDownList component as mentioned in the following ways.